src/lib/storage.ts

```diff
@@ -1,13 +1,22 @@
 import { readFileSync } from 'fs';
+import * as path from 'path';
 import { ConfigLoaderOptions } from './loader';
 const fromPairs = (pairs: Array<[keyof any, any]>) => Object.assign({}, ...Array.from(pairs, ([k, v]) => ({ [k]: v })));
 const { keys } = Object;
 
+const loadFile = (filePath?: string) => {
+    try {
+        return filePath ? JSON.parse(readFileSync(filePath, 'utf-8')) : {};
+    } catch (error) {
+        throw new Error(`Missing or invalid config file \`${filePath}\` at ${path.join(path.resolve('.'), filePath!)}`);
```

Comment:
```suggestion
        throw new Error(`Missing or invalid config file \`${filePath}\` at ${path.join(path.resolve('.'), filePath!)}. Update the path via \`defaultConfig\` option, or unset it to diable it.`);
```

Reply:
`...or fix json formatting`

But we dont know if its the defaultConfig or userConfigPath. Logic is the same for both

Reply:
:roll_eyes: